frontend/src/components/common/Filter/components/FilterSearch/index.tsx

```diff
@@ -13,13 +15,38 @@ export default function FilterSearch({
   searchValue,
   setSearchValue,
 }: Props): JSX.Element {
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  /**
+   * Clears search value and sets focus on the search input element.
+   */
+  const onClearSearch = (): void => {
+    setSearchValue("");
+    inputRef?.current?.focus();
+  };
+
   return (
     <ViewSearch
       autoFocus
       className={className}
+      inputRef={inputRef}
       leftIcon={IconNames.SEARCH}
       onChange={(changeEvent) => setSearchValue(changeEvent.target.value)}
       placeholder="Search"
+      rightElement={
+        searchValue ? (
+          <ClearIconButton
+            onClick={onClearSearch}
+            {...{
+              // TODO(cc) move this back to explicit prop={value} after upgrading SDS to enable type checking again
```

Comment:
Newer versions require SDS prop `sdsSize` for IconButton.

Reply:
Thanks for the comment! I'm working on upgrading to the latest SDS now. Hopefully can get out a PR next week 🚀 !

Reply:
I'll make sure to update the code here in my SDS upgrade PR too 👍

Reply:
Thanks so much @tihuan! So pumped for the latest SDS - it looks 🤩 !